src/Microsoft.Health.Fhir.Shared.Api/Features/Resources/Bundle/BundleHandler.cs

```diff
@@ -151,6 +153,8 @@ private static List<OperationOutcomeIssue> GetOperationOutcomeIssues(List<Operat
 
         private async Task FillRequestLists(List<Hl7.Fhir.Model.Bundle.EntryComponent> bundleEntries)
         {
+            int order = 0;
+            _requestCount = bundleEntries.Count;
             foreach (Hl7.Fhir.Model.Bundle.EntryComponent entry in bundleEntries)
             {
                 if (entry.Request.Method == null)
```

Comment:
Would this be a problem with the order counting? I don't know what the response looks like but assume you have an incoming bundle of GET, POST, empty, DELETE, when it encounter the empty request, it skips is but does not increment the order, what would the response look like?

Reply:
good point.. so as of now let's say request has 10 entries(with one empty entry), then response has 9 entries as its not appending anything for the empty one.

Reply:
I don't think that would be a desired behavior. We should either reject the call altogether (because it contains invalid entry) or return the empty entry where it is so that the caller can correlate their request to the response.

Reply:
Since empty entry is probably invalid, one potential fix is to return an OperationOutcome indicating that it was a bad request for this entry.